src/Particle/LongRange/KContainer.h

```diff
@@ -72,16 +72,21 @@ class KContainer : public QMCTraits
    * @param kc cutoff radius in the K
    * @param useSphere if true, use the |K|
    */
-  void updateKLists(const ParticleLayout& lattice, RealType kc, unsigned ndim, bool useSphere = true);
+  void updateKLists(const ParticleLayout& lattice,
+                    RealType kc,
+                    unsigned ndim,
+                    const PosType& twist = PosType(),
```

Comment:
Still missing documentation for twist. Say something about how twist affects KLists.